external/LibraryThemeStyles/Module.php

```
@@ -0,0 +1,387 @@
+                if ($debug) { $messenger->addSuccess('Debug: stored defaults sample: ' . substr(json_encode($current), 0, 300) . '...'); }
+            } else {
+                $messenger->addWarning('No action selected.');
+            }
+        } catch (\Throwable $e) {
+            error_log('[LibraryThemeStyles] ERROR: ' . $e->getMessage());
+            $messenger->addError('Error: ' . $e->getMessage());
+        }
+        return true;
+    }
+
+    private function applyPresetToThemeSettings($api, $siteSettings, ?string $siteSlug, string $themeKey, string $preset): array
```

PHPMD (2.15.0)

Avoid unused parameters such as '$themeKey'. (undefined)

(UnusedFormalParameter)